initializers/view.go

```diff
@@ -18,6 +19,7 @@ func SetUpTemplateFunction() {
 		"sentence_case": helpers.ToSentenceCase,
 		"render_file":   renderFile,
 		"render_icon":   renderIcon,
+		"format_datetime": formatDateTime,
```

Comment:
As there are functions(calling from template), I think we should use our function name convention. What do you think?
```suggestion
		"formatDateTime": formatDateTime,
```

Reply:
@junan Actually I'm using `snake_case` with lower case to follow the existing [template function names](https://beego.me/docs/mvc/view/template.md#template-functions)